ISX/ISX/CustomSuggestionProductsCell.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  CustomSuggestionProductsCell.swift
+//  ISX
+//
+//  Created by Rosyl Budike on 03-10-17.
+//  Copyright © 2017 Maren Osnabrug. All rights reserved.
+//
+
+import UIKit
+
+class CustomSuggestionProductsCell: UICollectionViewCell {
+
+    @IBOutlet weak var suggestionImage: UIImageView!
+    @IBOutlet weak var suggestionViewText: UIView!
```

Comment:
this should be view/containerView or textContainer/labelContainer, not View Text